### PR TITLE
DOCS-362 - Log Search Behavior Insights refactor

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request refactors Log Search Behavior Insights content per @banant 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

https://sumologic.atlassian.net/browse/DOCS-362

### CODE REVIEW

File: docs/get-started/ai-machine-learning.md [hunk 1]
```diff
@@ -49,11 +49,11 @@ With Copilot, you can effortlessly investigate complex issues without writing in
...
+LogCompare simplifies log analysis by enabling easy comparison of log data from different time periods to detect changes or anomalies, facilitating troubleshooting and root cause discovery. By automatically running delta analysis, LogCompare streamlines the process, allowing users to identify significant alterations in log patterns efficiently. Utilizing baseline and target queries, LogCompare clusters logs into patterns and compares them based on the significance of change, providing insights into deviations over time. With intuitive actions like promoting, demoting, and splitting signatures, users can refine their analysis and focus on relevant patterns, ultimately enhancing decision-making and threat detection capabilities. Additionally, LogCompare supports alerts and scheduled searches to notify users of new signatures or significant changes, ensuring proactive monitoring and response to evolving log data [Learn more](/docs/search/behavior-insights/logcompare).
```
Comment (jpipkin1) [inline docs/get-started/ai-machine-learning.md:56]:
```suggestion
LogCompare simplifies log analysis by enabling easy comparison of log data from different time periods to detect changes or anomalies, facilitating troubleshooting and root cause discovery. By automatically running delta analysis, LogCompare streamlines the process, allowing users to identify significant alterations in log patterns efficiently. Utilizing baseline and target queries, LogCompare clusters logs into patterns and compares them based on the significance of change, providing insights into deviations over time. With intuitive actions like promoting, demoting, and splitting signatures, users can refine their analysis and focus on relevant patterns, ultimately enhancing decision-making and threat detection capabilities. Additionally, LogCompare supports alerts and scheduled searches to notify users of new signatures or significant changes, ensuring proactive monitoring and response to evolving log data. [Learn more](/docs/search/behavior-insights/logcompare).
```

File: docs/search/behavior-insights/index.md [hunk 2]
```diff
@@ -12,9 +12,6 @@ Behavior Insights encompasses three log search operators to accelerate insights,
 
 Modeled after our LogReduce log summarization feature, the LogReduce Values and LogReduce Keys operators cluster logs based on their structure or pattern and activity content respectively.
 
-
-## Guide contents
-
 In this section, we'll introduce the following concepts:
```
Comment (jpipkin1) [inline docs/search/behavior-insights/index.md:16]: The cards need to be updated to show the current articles in the section.

File: docs/search/behavior-insights/logreduce/index.md [hunk 3]
```diff
@@ -32,25 +32,25 @@ In this section, we'll introduce the following concepts:
 <div className="box-wrapper" >
```
Comment (jpipkin1) [inline docs/search/behavior-insights/logreduce/index.md:32]: The cards need to be updated to show the current articles in the section.

File: docs/search/index.md [hunk 4]
```diff
@@ -37,13 +37,13 @@ In this section, we'll introduce the following concepts:
 </div>
 <div className="box smallbox card">
```
Comment (jpipkin1) [inline docs/search/index.md:38]: The cards need to be updated to show the current articles in the section.

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
Nice work, @kimsauce! I suggest you look at all the index articles affected by the move and make sure that the cards show the correct child articles.

File: docs/search/behavior-insights/logreduce/index.md (hunk 3 above)
Comment (kimsauce) [reply docs/search/behavior-insights/logreduce/index.md:32]: great catch!

**Comment by kimsauce** [comment]
> Nice work, @kimsauce! I suggest you look at all the index articles affected by the move and make sure that the cards show the correct child articles.

Thank you for this catch @jpipkin1  ! Just made the fixes - can you re-review pls?

**Comment by jpipkin1** [comment]
Your fixes look good. I made one more tweak (see commit "[Move 'Lookup Tables' in index](https://github.com/SumoLogic/sumologic-documentation/pull/4869/commits/f2e51ebf828fc6a8cb1654f15f8347df427270b6)").

**Review by jpipkin1 — APPROVED** [review]

**Pull request merged**